common/components/tag/_tag.scss

```diff
@@ -10,8 +12,8 @@
   outline: 2px solid transparent;
   outline-offset: -2px;
 
-  color: govuk-colour("white");
-  background-color: govuk-colour("blue");
+  color: govuk-colour('white');
```

Comment:
I'm guessing this might be some autosave feature on your IDE with the quotes?

It would be good if we keep them as is and consistent with the rest of the files for now.

Reply:
I'm using VSCode with the ESlint and Prettier ESLint extensions. Pretty sure it didn't reformat these files like that before. - maybe the rules have changed a bit? Anyhow, the singles are doubles again